python/ray/serve/_private/controller.py

```diff
@@ -212,6 +228,22 @@ async def __init__(
             description="The number of times that controller has started.",
         ).inc()
 
+    def reconfigure_system_logging_config(self, logging_config: LoggingConfig):
+        if self.logging_config.version == logging_config.version:
+            return
+        self.kv.put(LOGGING_CONFIG_CHECKPOINT_KEY, pickle.dumps(logging_config))
+        self.logging_config = logging_config
+
+        self.long_poll_host.notify_changed(
+            LongPollNamespace.SYSTEM_LOGGING_CONFIG,
+            logging_config,
+        )
```

Comment:
I believe this also needs to be called when we load the logging config in the constructor. Another reason to unify the codepaths.